src/lu.cpp

```diff
@@ -195,6 +196,7 @@ JNIEXPORT void JNICALL Java_com_powsybl_math_matrix_SparseLUDecomposition_update
                 throw std::runtime_error("klu_factor error " + context.error());
             }
         }
+        return context.common.rgrowth;
```

Comment:
It's a bit strange to return this, as if you did a whole factorization, you're returning a rgrowth which corresponds to the failed refactorization. But maybe that's exactly what you want to know?

Reply:
I would like 2 things:
 - monitor whether the matrix has been factorized or refactorized (no way with current code, because there is no log on c++ side and nothin if returned to Java)
 - figure out what is the value of rgrowth. I have no idea what is a correct growth threshold. I guess it depends on the network (size, control, meshing,  etc). So by logging (on Java side) the value it would be simpler for me to check how far it is usually from the threshold and what could be a correct value.